src/NzbDrone.Core/Download/Clients/uTorrent/UTorrent.cs

```diff
@@ -42,29 +42,26 @@ public override void MarkItemAsImported(DownloadClientItem downloadClientItem)
             if (Settings.TvImportedCategory.IsNotNullOrWhiteSpace() &&
                 Settings.TvImportedCategory != Settings.TvCategory)
             {
-                try
-                {
-                    _proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
+                _proxy.SetTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvImportedCategory, Settings);
 
-                    if (Settings.TvCategory.IsNotNullOrWhiteSpace())
-                    {
-                        _proxy.RemoveTorrentLabel(downloadClientItem.DownloadId.ToLower(), Settings.TvCategory, Settings);
-                    }
-                }
-                catch (Exception ex)
```

Comment:
I didn't want to just catch any exception here, but I couldn't actually find a scenario where setting a label would throw an exception as uTorrent will just create a label on set if it doesn't exist. If anything goes wrong here, it's likely to be a connection to uTorrent issue that may be better served being labeled appropriately as an error in DownloadEventHub